demo/schemas/user-collection.js

```diff
@@ -7,6 +7,17 @@ module.exports = {
     description: 'A collection of users.',
     type: 'object',
     properties: {
+        limit: {
```

Comment:
I think we need to be able to define the default value of the query string parameter in the schema as well and have validation use the default if it doesn't find anything.